docs/src/main/tut/testing.md

```diff
@@ -135,11 +130,25 @@ val foundNone: UserRepo[IO] = new UserRepo[IO] {
   def find(id: String): IO[Option[User]] = IO.pure(None)
 } 
 
-val foundNone = service(success).run(
+val response: IO[Response[IO]] = service[IO](success).orNotFound.run(
   Request(method = Method.GET, uri = Uri.uri("/user/not-used") )
 )
 
-assert[IO](result, Status.NotFound, None)
+check(response, Status.NotFound, None)
+```
+
+Finally, let's pass a `Request` for which our service does not handle it:  
```

Comment:
Diction here is a bit awkward.  "Finally, let's pass a`Request` which our service does not handle.", maybe?

Reply:
Yes - that's not clear at all as-is. I pushed up a change per your suggestion.